docs/cloud/runtime-troubleshooting-with-functions.md

```diff
@@ -16,8 +16,8 @@ These routines can be used to retrieve additional information to help you troubl
 ### Prerequisites
 
 The following is required to be able to run Functions from Netdata Cloud.
-* Have at least one capable node claimed to one of your Spaces (node on a version higher than `v1.37`)
-* Ensure that node has the collector that exposes the function enabled
+* Have at least one capable node claimed to one of your Spaces where node needs to be on a version higher than `v1.37`
```

Comment:
```suggestion
* At least one of the nodes claimed to your Space should be on a Netdata agent version higher than `v1.371.
```

Reply:
a typo v1.371 => v1.37.1

Reply:
I'll take in @shyamvalsan 's suggestion and will take care of the typo. @ilyam8 
thanks for checking that, I thought the functions capability had been introduced on `v1.37`

Reply:
pushed a commit for this